roottest/root/io/compression/CMakeLists.txt

```diff
@@ -1,7 +1,47 @@
-#-------------------------------------------------------------------------------
-#
-# Placeholder file to translate the tests to the new CTest system. Meanwhile we
-# define a CTest test that calls 'make' in ${CMAKE_CURRENT_SOURCE_DIR}
-#
-#-------------------------------------------------------------------------------
-ROOTTEST_ADD_OLDTEST()
+ROOT_GENERATE_DICTIONARY(G__IoCompressionGeneration
+  ${CMAKE_CURRENT_SOURCE_DIR}/Event.h
+  LINKDEF ${CMAKE_CURRENT_SOURCE_DIR}/EventLinkDef.h)
+
+ROOT_LINKER_LIBRARY(IoCompressionGeneration
+  ${CMAKE_CURRENT_SOURCE_DIR}/IoCompressionGeneration.cxx
+  G__IoCompressionGeneration.cxx
+  LIBRARIES Core Tree Hist MathCore Physics Graf Matrix)
+
+if(MSVC)
+  add_custom_command(TARGET IoCompressionGeneration POST_BUILD
+    COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_BINARY_DIR}/$<CONFIG>/libIoCompressionGeneration.dll
+                                     ${CMAKE_CURRENT_BINARY_DIR}/libIoCompressionGeneration.dll
+    COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_BINARY_DIR}/$<CONFIG>/libIoCompressionGeneration.lib
+                                     ${CMAKE_CURRENT_BINARY_DIR}/libIoCompressionGeneration.lib)
+endif()
```

Comment:
From https://github.com/root-project/roottest/pull/774#discussion_r2020252704:

@bellenot Is that an okay solution?

Reply:
> From [root-project/roottest#774 (comment)](https://github.com/root-project/roottest/pull/774#discussion_r2020252704):
> 
> @bellenot Is that an okay solution?

Almost. It should be:
```
if(MSVC AND NOT CMAKE_GENERATOR MATCHES Ninja)
```

Reply:
I guess, problem can be resolved using `ROOTTEST_GENERATE_DICTIONARY` macro.